Secret-agency-service/src/main/java/cz/fi/muni/pa165/secretagency/service/facade/AgentFacadeImpl.java

```diff
@@ -58,6 +59,13 @@ public Long createAgent(AgentCreateDTO agentCreateDTO) {
         return savedAgent.getId();
     }
 
+    @Override
+    public AgentDTO editAgent(AgentUpdateDTO agentUpdateDTO) {
+        Agent mappedAgent = beanMappingService.mapTo(agentUpdateDTO, Agent.class);
```

Comment:
Podle mě se ti nenamapuje depertmentId na Department, ověřoval jsi to?

Reply:
uz je to ok